src/styles/page-template/index.md.njk

```diff
@@ -113,6 +113,10 @@ Variables can be set with:
       <td class="govuk-table__cell">mainClasses</td>
       <td class="govuk-table__cell">Add a class to the <code>&lt;main&gt;</code> element. </td>
     </tr>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">assetUrl</td>
+      <td class="govuk-table__cell">Specify an absolute URL for assets. Currently used on <code>&lt;meta property="og:image"&gt;</code> tag, which needs to have an absolute url.</td>
```

Comment:
"Specify an absolute URL for assets" Makes it seem as though this is set for all assets, when it's only used for the open graph image?